README.md

```diff
@@ -26,12 +26,12 @@ cd sites/default/files/civicrm/ext
 # Download and enable the extension
 git clone https://github.com/civicrm/org.civicrm.shoreditch
 cv en org.civicrm.shoreditch
-
-# Enable the "CiviCRM Theme" module and set "Seven" as the Civi admin theme
-drush en -y civicrmtheme
-drush vset civicrmtheme_theme_admin seven
-drush cc civicrm
 ```
+
+### Companion theme
```

Comment:
We might change the text in the section to something like
```
### Companion theme for other CMS's
Drupal 7 - [CiviAdmin Companion Drupal theme](https://github.com/compucorp/shoreditch-companion-d7-theme)

Other CMS's are not supported yet.
```

This way the above block is not necessary anymore, as this serves both. And in future we can add other CMS's theme link when they are ready.